Content.Shared/Objectives/Systems/ObjectiveSystem.cs

```diff
@@ -0,0 +1,135 @@
+using Content.Shared.Mind;
+using Content.Shared.Objectives;
+using Content.Shared.Objectives.Components;
+using Robust.Shared.Utility;
+
+namespace Content.Shared.Objectives.Systems;
+
+/// <summary>
+/// Provides API for creating and interacting with objectives.
+/// Adds default info from <see cref="ObjectiveComponent"/> to <see cref="ObjectiveGetInfoEvent"/>.
+/// </summary>
+public sealed class ObjectiveSystem : EntitySystem
```

Comment:
This should really be merged with ObjectivesSystem, or just any option other than having both ObjectiveSystem and Objective**s**System.

Reply:
yes it is unfortunate